app.js

```diff
@@ -50,6 +50,7 @@ const Sequence = require('./helpers/sequence.js');
 const httpApi = require('./helpers/http_api.js');
 // eslint-disable-next-line import/order
 const swaggerHelper = require('./helpers/swagger');
+const createStorage = require('./storage');
```

Comment:
We are doing the `require` inside each method for the other ones.
Maybe use the same approach here?

Reply:
That was a bad approach, don't know why it was used.

Reply:
I agree but imo we should either follow the approach or fix it all together.